lib/config.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
```

Comment:
Please add a detailed comment about what this does and how it interacts with other key classes.

Reply:
there is a class description now